internal/flags.go

```diff
@@ -75,6 +76,23 @@ func objectKindSupportsLabelsFlag(kind manifest.Kind) bool {
 	return ok
 }
 
+var sloSupportingKinds = map[manifest.Kind]struct{}{
+	manifest.KindBudgetAdjustment: {},
+}
+
+func objectKindSupportsProjectSloFlag(kind manifest.Kind) bool {
+	_, ok := sloSupportingKinds[kind]
+	return ok
+}
+
+func registerProjectSloFlag(cmd *cobra.Command, storeSloIn, storeProjectIn *string) {
+	cmd.Flags().StringVarP(storeSloIn, "filters-slo-name", "", "",
+		`Filter resource by slo name. Example:	my-sample-slo-name `)
+	cmd.Flags().StringVarP(storeProjectIn, "filters-slo-project", "", "",
```

Comment:
Why not use the predefined -p/--project flag we're using in other get commands?
I know it doesn't refer to `metadata.project` in this case, but I think It will be more intuitive (at least for me) to use the familiar flag, you can extend the flag description to make it more clear, which field is filtered